### PR TITLE
Feature/718 proposal warnings

### PR DESCRIPTION
**Pull request by skuhlmann**
## GitHub Issue

fixes https://github.com/HausDAO/daohaus-monorepo/issues/718

## Changes

- new warning component and helpers for content

## Packages Added

n/a

## Checks

Before making your PR, please check the following:

- [x] Critical lint errors are resolved
- [x] App runs locally
- [x] App builds locally (run the build command for *any impacted package* and check for any errors before the PR)

### CODE REVIEW

File: apps/core-app/src/utils/constants.ts [hunk 1]
```diff
@@ -14,6 +14,55 @@ export const PROPOSAL_FILTERS: { [key: string]: string } = {
...
+    'This proposal is adding a Shaman with Share Management permissions to the DAO. Be sure to look at the contract details for more information.',
+  UPDATE_GOV_SETTINGS:
+    'This proposal will adjust the governanace settings for the DAO. Be sure to look at the contract details for more information.',
+  TOKEN_SETTINGS:
+    'This proposal will adjust the token transferability settings for the DAO. Be sure to look at the contract details for more information.',
+  TOKENS_FOR_SHARES:
+    'This proposal mints or removes tokens for DAO membership. Be sure to look at the contract details for more information.',
+  GUILDKICK:
+    'This proposal mints or removes tokens for DAO membership. Be sure to look at the contract details for more information.',
+  ERROR_DECODE:
```
Comment (brossetti1) [inline apps/core-app/src/utils/constants.ts:64]: just a suggestion - maybe `ERROR_CANNOT_DECODE` is more clear?

Comment (skuhlmann) [reply apps/core-app/src/utils/constants.ts]: updated

File: apps/core-app/src/utils/constants.ts [hunk 2]
```diff
@@ -14,6 +14,55 @@ export const PROPOSAL_FILTERS: { [key: string]: string } = {
...
+  AddShaman = 'ADD_SHAMAN',
+  TransferErc20 = 'TRANSFER_ERC20',
+  TransferNetworkToken = 'TRANSFER_NETWORK_TOKEN',
+  UpdateGovSettings = 'UPDATE_GOV_SETTINGS',
+  UpdateTokenSettings = 'TOKEN_SETTINGS',
+  TokensForShares = 'TOKENS_FOR_SHARES',
+  GuildKick = 'GUILDKICK',
+}
+
+export const PROPOSAL_TYPE_LABELS: { [key: string]: string } = {
```
Comment (jordanlesich) [inline apps/core-app/src/utils/constants.ts:33]: Record<keyof type ProposalTypeIds, string>

**Review by jordanlesich — APPROVED** [review]

File: apps/core-app/src/utils/constants.ts (hunk 2 above)
Comment (skuhlmann) [reply apps/core-app/src/utils/constants.ts:33]: having trouble with this.

**Pull request merged**